pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1216,7 +1183,46 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListenerForPortOrUDS(l
 	}
 }
 
-// buildSidecarInboundMgmtListeners creates inbound TCP only listeners for the management ports on
+// Deprecated: To be removed from code in 1.2. Likely redudant prior to 1.1 and turned off
```

Comment:
why is this deprecated? This is where we build the listeners for all the probe ports that people provide in k8s

Reply:
I asked costin about this, the suggestion was with the revised mechanism for health-checking this would be unnecessary

Reply:
If you mean the app probe rewrite, then it is disabled by default.